charts/illumidesk/templates/grader_setup_service.yaml

```diff
@@ -101,13 +101,25 @@ spec:
       containers:
         - name: grader-setup-container
           image: {{ .Values.graderSetupService.graderSetupImage }}
-          imagePullPolicy: Always
+          resources:
+            requests:
+              cpu: {{ .Values.graderSetupService.graderCPU | default "0.2" }}
+              memory: {{ .Values.graderSetupService.graderMEM | default "200Mi" }}
+            limits:
+              cpu: {{ .Values.graderSetupService.graderCPU | default "0.2" }}
+              memory: {{ .Values.graderSetupService.graderMEM | default "200Mi" }}
+          imagePullPolicy: IfNotPresent
           ports:
             - containerPort: 8000
           env:
             - name: GRADER_IMAGE_NAME
-              value: {{ .Values.graderSetupService.graderImage }}
-              
+              value: {{ .Values.graderSetupService.graderSpawnerImage | default "illumidesk/grader-notebook:latest" }}
+            - name: GRADER_CPU
```

Comment:
There's usually a 1:2 ration for CPU/RAM. Let's use 0.5 CPU as the default if we use 1Gi for memory as the default.

Reply:
Updated Grader Setup and Grader resource values and envars